test/e2e_node/kubelet_test.go

```diff
@@ -44,12 +44,14 @@ var _ = Describe("Kubelet", func() {
 	})
 
 	Describe("pod scheduling", func() {
+		namespace := "pod-scheduling"
```

Comment:
I am not sure.  We can still delete the namespace without using a generated one right?  What other advantages does it provide?